py-polars/polars/utils/udfs.py

```diff
@@ -196,23 +182,23 @@ def rewritten_instructions(self) -> list[Instruction]:
         """The rewritten bytecode instructions from the function we are parsing."""
         return list(self._rewritten_instructions)
 
-    def to_expression(self, col: str) -> str | None:
-        """Translate postfix bytecode instructions to polars expression string."""
+    def to_expression(self, col: str, as_repr: bool = True) -> str | None:
```

Comment:
is `as_repr` only for debugging?

Reply:
> is `as_repr` only for debugging?

Yup - we will only ever return the string to the user warning, but it can be a convenience when testing/debugging.